MOxUnit/@MOxUnitFunctionHandleTestCase/run.m

```diff
@@ -16,6 +16,7 @@
 %
 % NNO 2015
 
+    start_tic = tic;
```

Comment:
Thanks for explaining the var = tic(); syntax, I was not aware of that.
Still, when using tic: does that mean that this function has to store all the start times for every single test? 
Instead, when using `clock`, the `etime` function also returns the time in seconds - so that is about as easy to use a `tic`, isn't it?